Gems/PhysX/Code/Tests/CharacterGameplayControllerTests.cpp

```diff
@@ -124,4 +133,33 @@ namespace PhysX
             EXPECT_THAT(expectedGravityMultiplier + i, testing::FloatEq(basis.m_gameplayController->GetGravityMultiplier()));
         }
     }
+
+    TEST_F(PhysXDefaultWorldTest, CharacterGameplayController_EntityFallsUnderGravity)
+    {
+        GameplayTestBasis basis(m_testSceneHandle, DefaultGravityMultiplier, DefaultGroundDetectionBoxHeight, DefaultFloorTransform);
+        auto* sceneInterface = AZ::Interface<AzPhysics::SceneInterface>::Get();
+        sceneInterface->RemoveSimulatedBody(basis.m_sceneHandle, basis.m_floor->m_bodyHandle);
+        
+        // Let scene run for a few moments so the entity can be manipulated by gravity from the gameplay component
+        auto startPosition = basis.m_controller->GetPosition();
+        
+        int timeStepCount = 10;
+        float totalTime = 0.0f;
+        float timeStep = AzPhysics::SystemConfiguration::DefaultFixedTimestep;
+        float distanceFell = 0.0f;
+
+        for (int i = 0; i < timeStepCount; i++)
+        {
+            basis.Update();
+            totalTime += timeStep;
+        }
+
+        auto endPosition = basis.m_controller->GetPosition();
+        
+        // calculate distance fallen (d = 0.5 * g * t^2)
+        distanceFell = 0.5f * (-10) * (totalTime * totalTime);
```

Comment:
`(-10.0f)` is a magic number. Please assign to a const variable.
````
const float gravity = -10.0f;
````
Question: Is -10 for gravity used on purpose for this test for some reason? If yes then please add a comment. If not maybe just use `AzPhysics::DefaultGravity.GetZ()` instead.

Reply:
I was using -10 for debugging and mistakenly removed the code to get the gravity from the scene using `basis.m_testScene->GetGravity();` and using that object's z value.